openapi3/refs.go

```diff
@@ -41,11 +62,13 @@ func (value ExampleRef) String() string {
 	return fmt.Sprintf("%s:ExampleRef", value.Ref)
 }
 
-func (value ExampleRef) Resolved() bool { return !value.EmptyRef() && value.Value != nil }
+func (value ExampleRef) Resolved() bool { return value.Ref != "" && value.Value != nil }
 func (value ExampleRef) IsRef() bool    { return value.Ref != "" }
 func (value ExampleRef) IsValue() bool  { return value.Ref == "" && value.Value != nil }
 func (value ExampleRef) IsValid() bool  { return value.Ref != "" || value.Value != nil }
 func (value ExampleRef) EmptyRef() bool { return value.Ref == "" }
+func (value ExampleRef) GetRef() string { return value.Ref }
+func (value *ExampleRef) ResetRef()     { value.Ref = "" }
```

Comment:
This is pedantic, but isn't this `ClearRef` and not really resetting to some former state?

Reply:
good call!